src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -1473,7 +1473,7 @@ PeerImp::onMessage(std::shared_ptr<protocol::TMReplayDeltaRequest> const& m)
         return;
     }
 
-    fee_.update(Resource::feeModerateBurdenPeer, "replay delta request");
+    fee_.update(Resource::feeModerateBurdenPeer, "replay_delta_request");
```

Comment:
`fee_`'s context is already initialized with "replay_delta_request", so you do not need to update the context here at all.
```suggestion
    fee_.fee = Resource::feeModerateBurdenPeer;
```